src/test/java/net/snowflake/client/core/SFArrowResultSetIT.java

```diff
@@ -131,7 +131,7 @@ public void testOnlyOfflineData() throws Throwable {
     }
     Schema schema = new Schema(fieldList);
 
-    // genreate 10 chunk of data
+    // generate 10 chunk of data
```

Comment:
Note that I'm not checking grammar, `chunk` should probably be `chunks`, but it's a test file, so I really wouldn't check the grammar of it anyway if I were to do a grammar pass.